examples/kubernetes/deployments/crds/authProviders/user-service.authProvider.yaml

```diff
@@ -1,13 +1,13 @@
 apiVersion: agogos.app/v1
 kind: GqlAuthProvider
 metadata:
-    name: user-service-auth provider
+    name: user-service-auth-provider
 spec:
-  type: oauth2/client_credentials
-  authority: "http://oidc-server-mock"
+  authType: oauth2/client_credentials
+  authority: "http://oidc-server-mock/connect/token"
   client_id: agogos_client_id
```

Comment:
authType is camelCased and client_id is with underscore. Better keep the camelCase standard for yaml